Source/AutoExtern.Test/coverlet.runsettings

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<RunSettings>
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="XPlat Code Coverage">
+        <Configuration>
+          <Exclude>[*]DAST.*,[*]DCOMP.*</Exclude>
```

Comment:
Because of the code blowup in the Dafny-to-C# compilation, instrumentation hangs and causes the CI to timeout. Since we ignore coverage of generated code anyways, this is safe.